commands/templates/svc/manifest-public.yml

```diff
@@ -23,8 +23,6 @@ http:
     success_codes: '200,301,302'
     healthy_threshold: 3
     unhealthy_threshold: 2
-    interval: 5s
-    timeout: 30s
     grace_period: 90s
```

Comment:
What is the default for grace periods?

Reply:
Zero.

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service_definition_parameters.html#sd-networkconfiguration